CHANGELOG.md

```diff
@@ -7,8 +7,39 @@ This file was introduced starting Kani 0.23.0, so it only contains changes from
 ## [0.37.0]
 
 ### Major Changes
+
 * Delete obsolete stubs for `Vec` and related options ([pull request](https://github.com/model-checking/kani/pull/2770) by @zhassan-aws) 
 
+## What's Changed
+
+* Function Contracts: Support for defining and checking `requires` and `ensures` clauses by @JustusAdam in https://github.com/model-checking/kani/pull/2655
+* Do not run performance benchmarks for tags by @adpaco-aws in https://github.com/model-checking/kani/pull/2745
```

Comment:
This can be removed

Reply:
done

Reply:
This is still in, but has to be removed.